<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameAchieve</class>
 <widget class="QWidget" name="GameAchieve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>826</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1440</width>
    <height>826</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1440</width>
    <height>826</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>方正姚体</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>游戏排行榜</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Picture.qrc">
    <normaloff>:/Material/Picture/aircraft.png</normaloff>:/Material/Picture/aircraft.png</iconset>
  </property>
  <widget class="QLabel" name="name_label_1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>359</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正姚体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>暂缺</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="name_label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>430</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正姚体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>暂缺</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="name_label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>501</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正姚体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>暂缺</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="name_label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>574</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正姚体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>暂缺</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="name_label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>647</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正姚体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>暂缺</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="score_label_3">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>501</y>
     <width>150</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正姚体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="score_label_4">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>574</y>
     <width>150</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正姚体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="score_label_1">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>359</y>
     <width>150</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正姚体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="score_label_2">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>430</y>
     <width>150</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正姚体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="score_label_5">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>647</y>
     <width>150</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正姚体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Picture.qrc"/>
 </resources>
 <connections/>
</ui>
